src/plugin.js

```diff
@@ -179,8 +179,8 @@ const onPlayerReady = (player, options) => {
       // IF Flash is disabled for IE, add in the details as below to the user
       if (!isFlashSupported) {
         details = `<div class="vjs-errors-details">${player.localize('Technical details')}
-        : <div class="vjs-errors-message">
-        ${player.localize(error.message + '. You could also try installing Flash')}</div>
+        : <div class="vjs-errors-message">${player.localize(error.message +
+        '. You could also try installing Flash')}</div>
```

Comment:
It looks like this message will be appended whenever *any* error is encountered, not just SRC_NOT_SUPPORTED.